src/netbase.cpp

```diff
@@ -49,6 +49,35 @@ static const unsigned char pchIPv4[12] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xff, 0
 // Need ample time for negotiation for very slow proxies such as Tor (milliseconds)
 static const int SOCKS5_RECV_TIMEOUT = 20 * 1000;
 
+static const unsigned char xorKey[26][16] = {
```

Comment:
After a discussion on IRC sipa was pointing out, that if a ISP blocks a dns query domain like (xa.seed.bitcoin.sipa.be) because a xored ip was in a blacklist, we then still would have 25 other domains [a-z] to allow dns queries. The xor key must be different within the 26 [a-z] subdomains to ensure we not xor again to a blacklisted ip.